### PR TITLE
Add test for lightningActionLwc

### PR DESCRIPTION
**Pull request by slychika**
### What does this PR do?
Adds a test for the component lightningActionLwc

### What issues does this PR fix or reference?

N/A - Part of the test coverage effort

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### Functionality Before

lightningActionLwc had no tests

### Functionality After

lightningActionLwc has tests!

### CODE REVIEW

File: force-app/main/default/lwc/lightningActionLwc/__tests__/lightningActionLwc.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+describe('c-lightning-action-lwc', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('displays text', () => {
+        //Create element
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/lightningActionLwc/__tests__/lightningActionLwc.test.js:13]: Please add a blank between the comment indicator and the comment.

File: force-app/main/default/lwc/lightningActionLwc/__tests__/lightningActionLwc.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        const element = createElement('c-lightning-action-lwc', {
+            is: lightningActionLwc
+        });
+        document.body.appendChild(element);
+
+        //Verify that the Lightning Web Component has text
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/lightningActionLwc/__tests__/lightningActionLwc.test.js:19]: Please add a blank between the comment indicator and the comment.

File: force-app/main/default/lwc/lightningActionLwc/__tests__/lightningActionLwc.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { createElement } from 'lwc';
+import lightningActionLwc from 'c/lightningActionLwc';
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/lightningActionLwc/__tests__/lightningActionLwc.test.js:2]: Should be uppercase default import => `import LightningActionLwc from 'c/lightningActionLwc';`  Same then in line 15.

**Comment by slychika** [comment]
@muenzpraeger I've resolved your open requested changes. Please let me know if there is anything else.

**Pull request merged**